ecosystem/nft-metadata-crawler/nft-metadata-crawler-utils/src/pubsub.rs

```diff
@@ -0,0 +1,86 @@
+// Copyright © Aptos Foundation
+
+use google_cloud_googleapis::pubsub::v1::{
+    publisher_client::PublisherClient, subscriber_client::SubscriberClient, AcknowledgeRequest,
+    PublishRequest, PublishResponse, PubsubMessage, PullRequest, PullResponse,
+};
+use tonic::{metadata::MetadataValue, transport::Channel, Request, Response};
+
+#[allow(deprecated)]
+pub async fn publish_uris(
```

Comment:
Let's add a comment on what this function does. Like instead of just saying "it publishes uris" we should explain in more detail what it does, e.g. what is the message thing and why is it an array.